index.html

```diff
@@ -51,6 +53,238 @@ <h3>LET’S CONNECT</h3>
+              <div class="canopy">
+                <div class="client">CANOPY</div>
+                <div class="counter"></div>
+                <div class="role">Back End Dev</div>
+                <div class="counter"></div>
+                <div class="year">2015</div>
+              </div>
+            </div>
+            <div class="description-box">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </div>
+            <div class="tag-box">
+              <ul class="tags">
+                <li class="tag">HTML</li>
+                <li class="tag">CSS</li>
+                <li class="tag">Javascript</li>
+              </ul>
+            </div>
+            <div class="action-box">
+              <div class="button">
+                <div class="btn-text">See Project</div>
+              </div>
+            </div>
+          </div>
+        </div>
+        <!-- End of third card -->
+        <div class="card-works">
+          <img
+            src="./images/work/multipost2.png"
+            alt="Tonic Image"
+            class="snapshot-portfolio"
+          />
+          <div class="left-block">
+            <div class="title-box">
+              <div class="project-title">Multi-post Stories</div>
+              <div class="canopy">
+                <div class="client">CANOPY</div>
+                <div class="counter"></div>
+                <div class="role">Back End Dev</div>
+                <div class="counter"></div>
+                <div class="year">2015</div>
+              </div>
+            </div>
+            <div class="description-box">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </div>
+            <div class="tag-box">
+              <ul class="tags">
+                <li class="tag">HTML</li>
+                <li class="tag">CSS</li>
+                <li class="tag">Javascript</li>
+              </ul>
+            </div>
+            <div class="action-box">
+              <div class="button">
+                <div class="btn-text">See Project</div>
+              </div>
+            </div>
+          </div>
+        </div>
+        <!-- End of fourth card -->
+      </section>
+      <!-- End of Work Section -->
+
+      <!-- ABOUT MYSELF SECTION -->
+      <section id="aboutmyself">
+        <div class="top">
+          <div class="h2">About <br />Myself</div>
+          <div class="para">
+            Hello I’m a software developer! I can help you build a product ,
+            feature or website Look through some of my work and experience! If
+            you like what you see and have a project you need coded, don’t
+            hestiate to contact me.
+          </div>
+          <div class="social-media">
+            <div class="supporting-text">
+              <div class="h3">LET’S CONNECT</div>
+            </div>
```

Comment:
According to the project requirements, this content should be wrapped inside a `hx` tag.

<img width="441" alt="Screen Shot 2022-05-27 at 17 42 20" src="https://user-images.githubusercontent.com/82426682/170797399-be05a7a3-dead-42e6-a0cc-8a647995890e.png">

Kindly fix this.